topo/topo_test.go

```diff
@@ -1050,7 +1050,7 @@ func TestResources(t *testing.T) {
 			if s := errdiff.Check(err, tt.wantErr); s != "" {
 				t.Errorf("Resources() unexpected err: %s", s)
 			}
-			if s := cmp.Diff(tt.want, got); s != "" {
```

Comment:
isn't this backwards? https://pkg.go.dev/github.com/google/go-cmp/cmp#example-Diff-Testing